src/view/draggable/draggable.jsx

```diff
@@ -92,14 +92,17 @@ export default class Draggable extends Component<Props> {
 
     const callbacks: DragHandleCallbacks = {
       onLift: this.onLift,
-      onMove: this.onMove,
-      onDrop: this.onDrop,
-      onCancel: this.onCancel,
-      onMoveUp: this.onMoveUp,
-      onMoveDown: this.onMoveDown,
-      onMoveRight: this.onMoveRight,
-      onMoveLeft: this.onMoveLeft,
-      onWindowScroll: this.onWindowScroll,
+      onMove: (clientSelection: Position) =>
```

Comment:
this was the main driver for the change: i wanted to remove more of the boilerplate in this file